recipes/m2w64-sysroot/meta.yaml

```diff
@@ -0,0 +1,165 @@
+{% set name = "m2w64-sysroot" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ GIT_DESCRIBE_TAG }}.r{{ GIT_DESCRIBE_NUMBER }}.g{{ GIT_DESCRIBE_HASH }}
```

Comment:
@ocefpaf, I wanted to go by the upstream versioning scheme which looks like `v12.0.0.r2.ggc561118da`. That's why I used `git_url` which gives us these jinja variables.